naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/resolvers/NaptimeResolver.scala

```diff
@@ -86,19 +98,26 @@ class NaptimeResolver extends DeferredResolver[SangriaGraphQlContext] with Stric
       .groupBy(_.resourceName)
       .map {
         case (resourceName, requests) =>
+          // Handle MultiGet and Non-Multigets differently, since multigets can be batched
           val (forwardRequests, reverseRequests) =
             requests.partition(_.arguments.exists(_._1 == "ids"))
 
-          // Handle MultiGet and Non-Multigets differently, since multigets can be batched
-          val forwardRelations =
-            fetchForwardRelations(forwardRequests, resourceName, ctx)
+          // partition forward requests by auth type
```

Comment:
nit: the comment should reflect that we need to make 1 call of `fetchForwardRelations` for each auth override type.